src/app/app.module.ts

```diff
@@ -152,6 +152,7 @@ const appRoutes: Routes = [
 		{ path: 'video', component: VideoSearchComponent },
 
 		{ path: 'discord', component: DiscordRedirect },
+        { path: 'games/:share_key', redirectTo: '/player/:share_key' },
```

Comment:
should this be in a component like the discord redirect?

Reply:
The discord redirect is because he's doing special logic to change the window.location.  This redirect is in app, so no need.

Reply:
Yeah, but I just thought I'd propose it for consistencies sake.

Reply:
I'd say nah.  If both the routes go to the same component, there's some duplicated code.  It's an extremely minor thing, but I think it's probably better to have the redirect chain so that if you for some reason change the component someday, you only have to do it in one place and it changes for both.